src/components/textfield.js

```diff
@@ -30,16 +30,24 @@ module.exports = function (app) {
             pattern: '',
             custom: '',
             customPrivate: false
+          },
```

Comment:
We can, but I don't really think it's crucial to this feature currently. In the future, if we support more than one condition, we need to build in the order of operations for each logical token. So theres more to that then simply changing an obj to array.